libs/react-helpers/src/lib/grid/date-range-filter.tsx

```diff
@@ -84,16 +85,18 @@ export const DateRangeFilter = forwardRef((props: IFilterParams, ref) => {
   };
 
   useEffect(() => {
-    props.filterChangedCallback();
-  }, [value]); //eslint-disable-line react-hooks/exhaustive-deps
+    if (props && props.filterChangedCallback) {
+      props?.filterChangedCallback();
+    }
+  }, [value, props]);
 
   const start = (value.start && toInputValue(value.start)) || '';
```

Comment:
chore (non blocking): I know this was already there but it looks like the `toInputValue` function has the same purpose as `formatForInput` in format/date.ts. Can we use that?